cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -0,0 +1,50 @@
+% OCI(1) OCI-IMAGE-TOOL User Manuals
+% OCI Community
+% JULY 2016
+# NAME
+oci-image-tool-unpack \- Unpack an image or image source layout
+
+# SYNOPSIS
+**oci-image-tool unpack** [src] [dest] [flags]
+
+# DESCRIPTION
+`oci-image-tool unpack` unpacks a given OCI image into a directory suitable to be used with `runc`.
```

Comment:
It's probably worth drawing a clear distinction between `unpack` and `create-runtime-bundle` here.  That distinction is mostly around rootfs-or-not and config translation.